pkg/qemu/qemu.go

```diff
@@ -38,6 +38,10 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// RecommendedQemuVersion is the recommemded QEMU version.
+// See commit 8c48cb3eef178914f1b8fffee324aab0c445fec5 .
```

Comment:
Is this reference still needed? We use `host: max` now instead of `host: cortex-a76`.

So maybe the comment should be updated that we use `-cpu max` by default, which requires QEMU 7.0.0?